addons/website_forum/static/src/js/website_forum.share.js

```diff
@@ -46,17 +46,22 @@ var ForumShare = publicWidget.registry.socialShare.extend({
      * @private
      */
     _render: function () {
-        var $question = this.$('article.question');
+        const questionEl = this.el.querySelector("article.question");
         if (!this.targetType) {
             this._super.apply(this, arguments);
         } else if (this.targetType === 'social-alert') {
-            $question.before(qweb.render('website.social_alert', {medias: this.socialList}));
+            const socialAlert = qweb.render("website.social_alert", {
+                    medias: this.socialList,
+                });
+            questionEl.insertAdjacentHTML("beforebegin", socialAlert);
```

Comment:
@msh-odoo Did you test that? When these conditions are met (a post hasn't answers since at least 10 days), an alert should appear when clicking on the post. It seems that in this case `this.el` is the "article.question" element, so questionEl is null.
![image](https://github.com/user-attachments/assets/9ef64e4c-6d30-4f05-beb8-5e42dd444148)

Could you please check, and test thoroughly before updating this PR. Thanks :pray:

Reply:
Furthermore, you have to remove this element as well in the destroy method.
![image](https://github.com/user-attachments/assets/26f55441-6893-4dd7-8765-dac9458c13e5)

Reply:
@SergeBayet ```questionEl``` is part of this.el, so in ```destroy``` method we are already setting ```el``` to ```null``` so no need to do anything here, I agree ```questionEl``` can be null in some case so I will add optional chaining there.

Reply:
@msh-odoo So, how do you explain that the alert element is still there when I click on edit??

Reply:
@qsm-odoo To produce and check whether modal is removed or not, you just need to create new forum post and you will get dialog, simply click on Edit button on top-right corner, which will open edit mode, in edit mode Dialog should be closed.

Regarding Alert, alert is rendered besides question and it will not be removed when we follow the above steps.

@SergeBayet The solution creates too much confusion, would you please explain to @qsm-odoo  what was the original solution and why you have suggested ```destroy``` method?

Reply:
Prior to test, ensure that what I said https://github.com/odoo/odoo/pull/181643#discussion_r1833821647 is fixed
```js
this.el && this.el.insertAdjacentHTML("beforebegin", socialAlert);
```
- Create a new post:
![image](https://github.com/user-attachments/assets/c4bac6f8-d260-4d5c-a976-36ff8e39a3fe)
- And post your question
![image](https://github.com/user-attachments/assets/5d838bd1-2f43-41f5-a8d9-48a41dd649ef)
- In the table 'forum_post', look for the post created and change the create_date, and write_date so it's more than 10 days berfore today. 
![image](https://github.com/user-attachments/assets/f8f0d205-3781-45a6-9bb4-edd38d635623)
- Click on the "Forum" tab in the menu to access the list of the posts, and click on the post created.
![image](https://github.com/user-attachments/assets/b772f011-c79c-4ab9-a8f5-a24997dd9bea)
- Now a social alert appears above the question:
![image](https://github.com/user-attachments/assets/07e1bcd1-1169-406b-9b87-89dd57e19b3c)
- Click on "Edit", the social alert is still there.

Reply:
@msh-odoo The suggested solution to use destroy is not mine but @loco-odoo 's https://github.com/odoo/odoo/pull/181643#issuecomment-2428551325

Reply:
@SergeBayet ```socialAlert``` is added before ```this.el``` so even if ```this.el``` is destroyed it will not remove ```socialAlert```

Reply:
I agree with @qsm-odoo.

1. I just tested version 15.0, and it seems that the feature was broken even before 15.0. As qsm said, we prefer to leave it as it is and remove the dead code from the master branch.

![image](https://github.com/user-attachments/assets/10a1eca0-aee2-4437-9ece-34ac4953b69f)

2. The feature in question has already been removed in version 17.0, so there is no point in spending more time on it.

@SergeBayet I have updated the PR and reverted the dead code as it was before. Now I guess PR is ready for the launch once the runbot is green.

Thanks : )  

cc: @msh-odoo

Reply:
@divy-odoo @qsm-odoo @msh-odoo  Indeed I've just tested in 15.0 and it doesn't work either. And no "bump" feature in 17.0... So I guess we all agree.